samwise.js

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.